docs/run_sims.rst

```diff
@@ -110,3 +110,53 @@ When the simulation and postprocessing is all complete, run ``buildstock_aws
 --clean your_project_file.yml``. This will clean up all the AWS resources that
 were created on your behalf to run the simulations. Your results will still be
 on S3 and queryable in Athena.
+
+Google Cloud Platform
+~~~~~~~~~~~~~~~~~~~~~
+
+Running a batch on GCP is done by calling the ``buildstock_gcp`` command line
+tool.
+
+.. command-output:: buildstock_gcp --help
+   :ellipsis: 0,8
+
+GCP Specific Project configuration
+..................................
+
+TODO: add more GCP configuration details here
+
+For the project to run on GCP, you will need to add a section to your config
+file, something like this:
+
+.. code-block:: yaml
+
+    gcp:
+      job_identifier: national01
+      gcp_project: myorg_project
+      region: us-central1
+      artifact_registry: buildstockbatch
+      gcs:
+        bucket: mybucket
+        prefix: national01_run01
+      use_spot: true
+      batch_array_size: 10000
+      notifications_email: your_email@somewhere.com
+
+See :ref:`gcp-config` for details.
+
+
+List existing jobs
+..................
+
+Run ``buildstock_gcp --list_jobs your_project_file.yml`` to see a list of all existing
+jobs matching the project specified. This can show you whether a previously-started job
+has completed or is still running.
+
+
+Cleaning up after yourself
+..........................
+
+When the simulation and postprocessing is all complete, run ``buildstock_gcp
+--clean your_project_file.yml``. This will clean up all the GCP resources that
```

Comment:
This is somewhat inaccurate (for lack of better word) for the version that we're developing since doing a run now also includes cleanup (i.e., you don't usually need to run cleanup separately). So, this should probably start with something more like: "Running a simulation automatically cleans up GCP resources when it completes. If you need to clean up manually (e.g., because the run did not complete) or cancel a run, run ``buildstock_gcp
--clean your_project_file.yml``..."

Reply:
I'm not sure that will stay true for all resources (for example, nothing is deleting the docker image you're uploading), but I'll add a TODO for now to make sure it's accurate when we actually implement this option.

Reply:
Ah, also, to clarify, my code for the GCP Batch job just kicks off the job, then exits. So the current code
```
batch.run_batch()
batch.process_results()
batch.clean()
```
won't actually work correctly, since cleanup needs to wait until the job actually finishes.

And since these jobs may take hours to run, I think that's the right model - the user shouldn't have to have to leave a script running locally for that long.